app/src/main/java/ar/com/wolox/android/training/ui/training/details/DetailsFragment.kt

```diff
@@ -0,0 +1,98 @@
+package ar.com.wolox.android.training.ui.training.details
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
+import android.net.Uri
+import android.os.Bundle
+import android.widget.Toast
+import ar.com.wolox.android.R
+import ar.com.wolox.android.training.model.EventMessage
+import ar.com.wolox.android.training.model.NewsItem
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import com.facebook.drawee.backends.pipeline.Fresco
+import kotlinx.android.synthetic.main.fragment_details.*
+import org.greenrobot.eventbus.EventBus
+import org.ocpsoft.prettytime.PrettyTime
+import java.util.Locale
+
+class DetailsFragment : WolmoFragment<DetailsPresenter>(), IDetailsView {
+
+    private lateinit var newsDetail: NewsItem
+    private var position: Int = -1
+    private var userId: Int = -1
+
+    override fun layout(): Int = R.layout.fragment_details
+
+    override fun handleArguments(arguments: Bundle?) = arguments != null
```

Comment:
On `handleArguments` you need to check if you're receiving all the arguments you want. Like:

```
override fun handleArguments(arguments: Bundle?) = 
               arguments != null && arguments!!.containsKey(KEY_NEWS_ITEM) && ...
```

Reply:
👀